integration/pkcs11/pkcs11_test.go

```diff
@@ -33,89 +33,106 @@ import (
 	"github.com/tedsuo/ifrit"
 )
 
-var _ = Describe("Configures PKCS#11 for peer and orderer", func() {
+var _ = Describe("PKCS11 enabled network", func() {
```

Comment:
Very minor but the _real_ name of the standard is PKCS #11 (see here https://en.wikipedia.org/wiki/PKCS_11). So while the previous string was incorrect due to the missing space, your's isn't correct either. If you are okay with changing I'd change it here and everywhere else.